qml/pages/ScreenBlank.qml

```diff
@@ -0,0 +1,34 @@
+import QtQuick 2.0
+import org.nemomobile.dbus 2.0
+
+// Source: https://together.jolla.com/question/93323/harbour-api-request-keep-screen-onprevent-screen-blanking/?answer=154926#post-id-154926
+
+Item {
+    property bool enabled: false
+
+    function request() {
+        var method = "req_display" + (enabled ? "" : "_cancel") + "_blanking_pause";
+        dbif.call(method, [])
+    }
+
+    onEnabledChanged: enabled && request()
```

Comment:
This makes request() not called if enabled becomes false.

Reply:
The display does blank eventually even with this code but it just takes much longer than usual.

Reply:
Oh, I left it running for quite a while and it did not blank, need to test again. :)

Good catch! I did not realize that the request is needed to turn the blanking also off. :sweat: